### PR TITLE
doc for pg-history-viewer

### PR DESCRIPTION
**Pull request by lbartoletti**
This PR adds a minimal documentation about the use of PG history viewer in QWAT
cc @ponceta

### CODE REVIEW

**Comment by haubourg** [comment]
LGTM. 

By the way, I saw that the documentation is still advising to use QGIS 2.18 . @ponceta is this OK if we raise version to QGIS 3.4 ?

File: installation-guide/workstation.rst [hunk 1]
```diff
@@ -24,11 +24,16 @@ A comfortable configuration would be:
 * Good video card (recent onboard Intel are ok, or dedicated NVidia/AMD)
 * Minimum requirement 2.14
 
-.. note:: 
+.. note::
   We recommend using the latest LTR version: 2.18.
```
Comment (ponceta) [inline installation-guide/workstation.rst:28]:
```suggestion
  We recommend using the latest LTR version: 3.4.
```

**Comment by ponceta** [comment]
LGTM. Thanks!

File: installation-guide/workstation.rst (hunk 1 above)
Comment (haubourg) [reply installation-guide/workstation.rst:28]: I think we should adress this in another PR, to keep commits unitary.

**Pull request merged**